lib/tests/testscripts/scripts/reporting/vte-data-requirements.xml

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TestScript xmlns="http://hl7.org/fhir">
+    <id value="vte-data-requirements"/>
+
+    <url value="http://example.com"/>
+    <name value="Data requirements test for VTE measure"/>
+    <status value="draft"/>
+    <date value="2019-04-04"/>
+    <publisher value="MITRE"/>
+    <contact>
+        <name value="Matthew Gramigna"/>
+        <telecom>
+            <system value="email"/>
+            <value value="mgramigna@mitre.org"/>
+            <use value="work"/>
+        </telecom>
+    </contact>
+    <description value="Tests using XML format to execute the $data-requirements operation operation. The destination server must support the $data-requirements operation on the Measure resource."/>
```

Comment:
Technically this TestScript tests the `JSON` formatted requests/responses for `$data-requirements`

Reply:
Whoops. Just pushed a small update to description wording.